src/static/Preview.html

```diff
@@ -12,14 +12,12 @@
     <link href="https://amp.azure.net/libs/amp/latest/skins/amp-default/azuremediaplayer.min.css" rel="stylesheet">
     <script src="https://amp.azure.net/libs/amp/latest/azuremediaplayer.min.js"></script>
     <!--*****END OF Azure Media Player scripts*****-->
+
     <!--*****START OF AMP plugin scripts*****-->
-    <script src="js/amp.preview.js"></script>
     <script src="js/ampPlayer.js"></script>
-    <script src="js/browserutils.js"></script>
-    <script src="js/globals.js"></script>
-    <script src="js/mpdutils.js"></script>
-    <script src="js/mseutils.js"></script>
+    <script src="js/bundle.js"></script>
```

Comment:
Is there any reason to keep `ampPlayer.js` file outside of the bundle file?

Reply:
I don't know why, but ampPlayer is unable to load the preview plugin if they are in the same file. https://amp.azure.net/libs/amp/latest/docs/index.html#writing-plugins has a specific note about the order in which scripts are loaded stating that plugins must be loaded after the AMP script

Reply:
Yup, Seems like the component (amp player) is first initialized in the browser, then the plugin is 'extra' and it has to be done after the instantiation.

Reply:
Got you, just curious.